src/main/java/decodes/launcher/LauncherFrame.java

```diff
@@ -2166,13 +2052,13 @@ protected void startProfileManager()
      */
     public Profile getSelectedProfile()
     {
-        return profileCombo == null ? null : (Profile)profileCombo.getSelectedItem();
+        return profileCombo == null ? launchProfile : (Profile)profileCombo.getSelectedItem();
     }
 
     private boolean trySendToProfileLauncher(String cmd)
     {
         profile = getSelectedProfile();
```

Comment:
profile here is null with a fresh XML database install.

The following error can be found with fresh XML install after clicking : Setup, or 'Decodes Database Editor' from the start_launcher main screen (without using profiles)

```text
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
        at decodes.launcher.LauncherFrame.trySendToProfileLauncher(LauncherFrame.java:2061)
        at decodes.launcher.LauncherFrame.setupPressed(LauncherFrame.java:962)
        at decodes.launcher.LauncherFrame$26.actionPerformed(LauncherFrame.java:677)
```

Reply:
Yep, the else clause in the profileCombo handling left the combo empty. I changed it to add the "launchProfile" item and set it as selected. Seems to behave now.